src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -1517,8 +1517,8 @@ InferenceProfiler::DetermineStatsModelVersion(
   }
 
   if (*status_model_version == -1) {
-    return cb::Error(
-        "failed to find the requested model version", pa::GENERIC_ERROR);
+    std::cerr << "WARNING: Failed to find the requested model version."
+              << std::endl;
```

Comment:
Have you confirmed that the program continues in an acceptable manner? Do we need to hard-code the `status_model_version` in this case to something other than `-1`?

Reply:
It was passing the unit tests which i thought was sufficient at the time. I think you are right tho and I need to take this thru a manual example. Let me do that and get back to you